app/settings/user-settings/user-settings.html

```diff
@@ -40,39 +40,66 @@ <h2 class="form-sheet-title" translate="settings.settings_list.user_settings">
             </div>
 
             <img class="field-logo" src="/img/hdx-logo.png" />
+            <form name="api">
+                <div
+                    class="form-field init required"
+                    ng-class="{'error': api.$invalid && api.$dirty,'success': !api.$invalid && api.$dirty}"
+                    ng-show="!hxlUserId"
```

Comment:
Can we rename `hxlUserId` to something meaningful? I spent a few minutes wonder what was going on till I realised its really a show/hide toggle, not the actually userId Model field

Reply:
Should the ng-show check include invalid and dirty checks here? (the same as for the api key below)

Reply:
Yes I think that's a good idea, I'll add it.